content/learning-paths/smartphones-and-mobile/debugging_with_mte_on_pixel8/how-to-4.md

```diff
@@ -0,0 +1,30 @@
+---
+title: Debugging in Android Studio with MTE
+weight: 5
+
+### FIXED, DO NOT MODIFY
+layout: learningpathall
+---
+
+## Debugging in Android Studio
+Your are now ready to debug the MTE Test application in Android Studio.
+Connect your Google Pixel 8 smatphone to your computer with the MTE Test project opened in Android Studio.
+You should see the device name in the box shown in the picture below.
+
+![alt-text-2](pictures/05_android_sutio_device_connected.png "Device recognized by Android Studio.")
+
+Press the *Debug* button as shown in the picture above to build and start debugging the application. Look at your device where the application is about to start. You will see the message below in your screen. Just wait until the application interface shopws up.
+
+![alt-text-2](pictures/06_waiting_for_debugger.png "Waiting for Debugger message.")
+
+Press the *Debug* button at the botton of the Android Studio window as shown in the picture below. This will open the debug console. The first time it could take around a minute for the console to show that Android Studio is "connected to the target VM".
```

Comment:
"bottom"! Also, technically that's a tab, not a button?
"Press the *Debug* tab at the bottom of the Android Studio window as shown in the picture below. This will open the debug console..."
Also: It can take a minute? Wow! :/